app/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -82,10 +75,13 @@ export default function CodeSnippet({
         startingLineNumber={startingLineNumber}
         lineProps={(lineNumber) => {
           let style: CSSProperties = {
-            opacity: colorMode === "dark" ? 0.9 : 1,
+            display: "block",
+            width: "100%",
```

Comment:
Please remove this style as well, the blocks seem to have 100% width by default. 
```suggestion
```